.travis.yml

```diff
@@ -1,18 +1,19 @@
 language: julia
 julia:
     - 0.6
+    # - 0.7
     - nightly
-matrix:
-  allow_failures:
-  - julia: nightly
+# matrix:
+#   allow_failures:
+#   - julia: nightly
 notifications:
```

Comment:
It might be better to do:

```
julia:
    - 0.7
    - nightly
matrix:
  allow_failures:
  - julia: nightly
```

I think the next release of ForwardDiff should be v0.7 only. We can backport future fixes to previous minor releases if needed. Trying to maintain a single codebase to support two separate Julia versions isn't worth it IMO.

Reply:
See comments here: https://github.com/JuliaDiff/DiffTests.jl/pull/3#discussion_r200322905
Arguably `nightly` is more important to test at this stage compared to `- 0.7` which is `0.7-beta`

Reply:
If a contributor makes a PR which passes `0.7` but not `nightly`, I'd merge it regardless, since I don't pass/fail that contributor's work based on unreleased/changing upstream behavior.

Downstream package CI really should not depend on unfixed upstream dependencies. If an upstream tool wants downstream tools to update, upstream can post new tags for the `0.7` target, and downstream can update accordingly.

Note that the allowing failures in the build matrix still runs the tests, so we can still keep our eye on it.

Reply:
Guess we have to wait for rc-1 then.